.gitignore

```diff
@@ -46,6 +46,9 @@ cscope*
 install/v2plugin/rootfs
 install/v2plugin/config.json
 
+# netcontain temporary build files
+scripts/netContain/bin
```

Comment:
Adds the bin directory to .gitignore. We really don't need to commit netplugin binaries to the repository. This was an issue before.